client/Lykke.Service.History.Client/IHistoryApi.cs

```diff
@@ -40,5 +45,28 @@ public interface IHistoryApi
         /// <returns></returns>
         [Get("/api/history/{walletId}/{id}")]
         Task<BaseHistoryModel> GetHistoryItemByIdAsync(Guid walletId, Guid id);
+
+        /// <summary>
+        /// Get trades by wallet id
+        /// </summary>
+        /// <param name="walletId"></param>
+        /// <param name="assetId"></param>
+        /// <param name="assetPairId"></param>
+        /// <param name="offset"></param>
+        /// <param name="limit"></param>
+        /// <param name="fromDt"></param>
+        /// <param name="toDt"></param>
+        /// <param name="tradeType"></param>
+        /// <returns></returns>
+        [Get("/api/history/trades")]
```

Comment:
I would move to dedicated interface, like "ITrades" with related path ("/api/trades"). Also, I think that IOrdersApi.GetTradesByOrderIdAsync should belong there.